docs/testing/debug-office-add-ins-on-ipad-and-mac.md

```diff
@@ -29,17 +29,19 @@ To start, open a terminal and set the `OfficeWebAddinDeveloperExtras` property f
 - `defaults write com.microsoft.Outlook OfficeWebAddinDeveloperExtras -bool true`
 
     > [!IMPORTANT]
-    > Mac App Store builds of Office do not support the `OfficeWebAddinDeveloperExtras` flag.
+    > Mac App Store builds of Office don't support the `OfficeWebAddinDeveloperExtras` flag.
 
 Then, open the Office application and [sideload your add-in](sideload-an-office-add-in-on-mac.md). Right-click the add-in and you should see an **Inspect Element** option in the context menu. Select that option and it will pop the Inspector, where you can set breakpoints and debug your add-in.
```

Comment:
Consider being less mouse-centric (per [style guide](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui))

```suggestion
Then, open the Office application and [sideload your add-in](sideload-an-office-add-in-on-mac.md). Select and hold (or right-click) the add-in and you should see an **Inspect Element** option in the context menu. Select that option and it will pop the Inspector, where you can set breakpoints and debug your add-in.
```